README.md

```
@@ -412,6 +413,138 @@ This artifact is the intended input for future drift-aggregation work (see issue
+  "trend": [
+    {"date": "2026-07-19", "total": 0, "drifted": 0, "drift_rate": 0.0},
+    {"date": "2026-07-20", "total": 1, "drifted": 0, "drift_rate": 0.0},
+    {"date": "2026-07-21", "total": 2, "drifted": 1, "drift_rate": 0.5},
+    {"date": "2026-07-22", "total": 1, "drifted": 0, "drift_rate": 0.0}
+  ]
+}
+```
+
+#### Example output (`--format text`)
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)